src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/ClassDefHandler.java

```diff
@@ -97,12 +102,38 @@ public void checkIndentation() {
     }
 
     /**
+     * Checks the {@code @interface ident} part of the trees.
+     * @param identAst Ident ast around which it is checked
+     */
+    private void checkAnnotatioDefIndent(final DetailAST identAst) {
+        final DetailAST atAst = getMainAst().findFirstToken(TokenTypes.AT);
+        final String atSubType = "@";
+        if (areOnSameLine(atAst, identAst) && !getIndent().isAcceptable(getLineStart(atAst))) {
```

Comment:
We should only validate tokens that start the line. You do no verification that this token, and the ones below, starts the line.
if you debug this, `isAcceptable` should never be called for non-start tokens.

Could we use `LineWrappingHandler` here with a supplied first indent to do this all for us?